engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/GetStackTraceTextNode.java

```diff
@@ -76,6 +77,11 @@ String printStackTrace(Throwable throwable) {
         var sourceLoc = errorFrame.getLocation().getEncapsulatingSourceSection();
         if (sourceLoc != null) {
           var path = sourceLoc.getSource().getPath();
+          try {
+            path = new File(sourceLoc.getSource().getURI()).getPath();
```

Comment:
This one is for you, @AdRiley - the stack traces will now point to Git locations of standard library sources when using `runEngineDistribution --run test/Base_Tests`.

Reply:
The comment is [Prefer getURI path when gathering location of a source](https://github.com/enso-org/enso/pull/11850/commits/d12bcee93a216581c613d2f00e37d3dd4e099549) provided as the commit message.

Reply:
The commit message will disappear once the commits will be squashed upon merge to `develop`. So it will be almost impossible to find it.

Reply:
I mean it is _technically possible_ to find it: I need to run Git blame to find the commit related to the PR, find that PR on github to see its original commit list and see if there's anything related. And in many cases there won't be much info anyway.

I can't expect anyone will do so much of archivist/historian work to learn about a single line of code. Why can't we just add a comment in the code in such case?